tasks/check-if-usns-are-applicable.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${0}" )" && pwd )"
+
+source "${SCRIPT_DIR}/usn-processing/usn-shared-functions.sh"
+
+packages_included_in_stemcell=false
+mapfile -t FOUND_USNS < usns/usns.json
+for usn in "${FOUND_USNS[@]}"; do
+  id=$(echo $usn | jq -r '.url' | cut -d '/' -f6 | cut -d '-' -f2,3)
+
+  usn_json="${PWD}/usn.json"
+  echo $usn > $usn_json
+  process_usns "${usn_json}" "usn-gh-json/usn"
+
+
+  if [ "$PACKAGE_INCLUDED_IN_STEMCELL" == true ]
```

Comment:
I think something is messed up in this loop where the first (or maybe last) usn to be checked doesn't get written to the usn-log.json, even if PACKAGES_INCLUDED_IN_STEMCELL is true